modules/core/client/directives/core.client.directives.js

```diff
@@ -713,4 +713,59 @@ angular.module('core')
         });
       }
     };
+  })
+  /**
+   *  A modal for exporting project data.
+   */
+  .directive('exportModal', function ($state, $uibModal, ProjectModel, $filter) {
+    return {
+      restrict: 'A',
+      scope: {
+        context: '=',
+        current: '=',
+        public: '=',
+        allowedRoles: '='
+      },
+      link: function (scope, element) {
+        element.on('click', function () {
+          $uibModal.open({
+            animation: true,
+            templateUrl: 'modules/core/client/views/export-modal.html',
+            controllerAs: 's',
+            // windowClass: 'export-modal',
+            size: 'md',
+            controller: function ($scope, $uibModalInstance, $timeout) {
+              var s = this;
+              s.isLoading = false;
+              $scope.url = null;
+              s.exportProjects = function () {
+                s.isLoading = true;
+                ProjectModel.exportProjects().then(function (csvData) {
+                  var blob = new Blob([csvData], {
+                    type: 'text/csv'
+                  });
+                  $scope.download = 'epic_project_data_' + $filter('date')(new Date(), 'yyyy-MM-dd') + '.csv';
```

Comment:
Is there a chance an Admin would check this twice in one day?  First time to see who needs updating, second to confirm the changes are how they wish them to be?

Reply:
Added hours/mins/secs to the filename